src/input/mpegts/tvhdhomerun/tvhdhomerun_frontend.c

```diff
@@ -454,6 +454,9 @@ static int tvhdhomerun_frontend_tune(tvhdhomerun_frontend_t *hfe, mpegts_mux_ins
           break;
       }
       break;
+    case DVB_TYPE_ISDB_T:
+      snprintf(channel_buf, sizeof(channel_buf), "auto:%u", dmc->dmc_fe_freq);
```

Comment:
Am I missing something or is this the default from below? There's no point in making it redundant, so this should be removed.

Reply:
I left a separated case for ISDB-T as I thought there might be some additional things specific for ISDB-T but you right and we can use the default case here. Will remove it.